Authentication/SignUp/View/SignupViewDelegate.swift

```diff
@@ -15,21 +15,21 @@ public protocol SignupViewDelegate {
     /** Palettes ued to configure all login view elements possible. */
     var colorPalette: ColorPaletteType { get }
     var fontPalette: FontPaletteType { get }
-    var termsAndServicesURL: NSURL { get }
+    var termsAndServicesURL: URL { get }
     
     /**
         Function to configure all view elements according to the palettes.
         It is called by the default signup view when rendered.
         
         By default, does nothing.
      */
-    func configureView(signupView: SignupViewType)
+    func configureView(_ signupView: SignupViewType)
```

Comment:
`configureSignupView(_ signupView: signupViewType)` would be the ideal naming based on conventions.